crates/builder-api/src/v0_1/builder.rs

```diff
@@ -50,6 +50,15 @@ pub enum BuildError {
     Error(String),
 }
 
+/// Enum to keep track on status of a transaction
+#[derive(Clone, Debug, Deserialize, Serialize, PartialEq)]
+pub enum TransactionStatus {
+    Pending,
+    Sequenced { block: u64 },
```

Comment:
In the issue [description](https://github.com/EspressoSystems/marketplace-builder-core/issues/175), looks like we also want the `offset` info for the `Sequenced` status. Is it intentially excluded here?

Reply:
Sorry I just realized you already said this at "This PR does not".